--- shorewall-4.5.21.4/shorewall-4.5.21.4/configfiles/shorewall.conf~	2013-11-16 18:07:23.000000000 +0200
+++ shorewall-4.5.21.4/shorewall-4.5.21.4/configfiles/shorewall.conf	2013-11-26 21:43:06.058797770 +0200
@@ -31,7 +31,7 @@
 
 LOGALLNEW=
 
-LOGFILE=/var/log/messages
+LOGFILE=/var/log/kernel
 
 LOGFORMAT="Shorewall:%s:%s:"
 
@@ -83,7 +83,7 @@
 
 RESTOREFILE=restore
 
-SHOREWALL_SHELL=/bin/sh
+SHOREWALL_SHELL=/bin/bash
 
 SUBSYSLOCK=/var/lock/subsys/shorewall
 
